dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/FrameworkSelection.ts

```diff
@@ -118,6 +119,7 @@ describe("The shared header of the framework pages should act as expected", { sc
        * @param header the h2 header to check for
        */
       function validateFrameworkPage(framework: DataTypeEnum, header: string): void {
+        cy.wait(5000);
```

Comment:
should this stay?

Reply:
Will check if this can be removed once the e2e tests are working again

Reply:
Tests worked without it, so i will delete the wait statement.

Reply:
ok